### DISCUSSION TITLE
Invoice creation

### DISCUSSION THREAD

**Discussion by pmuch78** [YetiForceCompany/YetiForceCRM#16940] [How to] [answered]
How can I add items to invoice? I've created the invoice, but it's empty - with no items.

**Comment by rskrzypczak** [comment, marked as answer]
Be more specific
![image](https://user-images.githubusercontent.com/10221999/223050276-cc3562d0-a66c-46c5-9dfd-4061631cf957.png)

**Comment by pmuch78** [comment]
W dniu 06-03-2023 o 08:54, rskrzypczak pisze:
>
> Be more specific
> image 
> <https://user-images.githubusercontent.com/10221999/223050276-cc3562d0-a66c-46c5-9dfd-4061631cf957.png>
>

I don't see it this way. In my view:

**Comment by rskrzypczak** [comment]
You probably changed your module type to basic.
![image](https://user-images.githubusercontent.com/10221999/223093003-6bb82e8c-5d46-43a3-9e7c-9bd50a6fda47.png)
Return to standard settings.

**Comment by pmuch78** [comment]
Thanks :-)

It was the reason.


Pawel


W dniu 06-03-2023 o 12:04, rskrzypczak pisze:
